homeassistant/components/point/alarm_control_panel.py

```diff
@@ -63,11 +63,14 @@ def _webhook_event(self, data, webhook):
         """Process new event from the webhook."""
         _type = data.get("event", {}).get("type")
         _device_id = data.get("event", {}).get("device_id")
-        if _device_id not in self._home["devices"] or _type not in EVENT_MAP:
+        _changed_by = data.get("event", {}).get("user_id")
+        if (
+            _device_id not in self._home["devices"] and _type not in EVENT_MAP
+        ) and _type != "alarm_silenced":  # alarm_silenced does not have device_id
```

Comment:
`"alarm_silenced"` is in `EVENT_MAP` so this last check isn't needed.

Reply:
Yes it is as "alarm_silenced does not have device_id"

Reply:
Then the very first check will catch that and short-circuit and the two following checks won't be checked. We combine the checks with `and`, so the first check that is falsy will short-circuit.

https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not

Reply:
Sorry, I missed the parenthesis was not wrapping the full statement.